src/wilson/dom.cljs

```diff
@@ -174,9 +174,15 @@
   (update elem 1 merge-attrs attrs))
 
 (defn with-class
-  "Adds a class to an element."
-  [cls elem]
-  (with-attrs {:class cls} elem))
+  "Adds a class to a component."
+  [cls component]
+  (if (vector? component)
```

Comment:
I see. Right now you can call it like this: `[with-class "yolo" thingy-component a b c]`. But I presume _forcing_ user to only use `()` makes a lot of sense